src/Pos/Wallet/Web/ClientTypes.hs

```diff
@@ -213,88 +205,82 @@ cPassPhraseToPassPhrase (CPassPhrase text) =
 -- | Wallet identifier
 data CWalletAddress = CWalletAddress
```

Comment:
Yep, I propose to do it later under CSM-249, since we're not being kicked regarding it )